cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,28 @@
 /// <reference types='cypress' />
+import MainPageObject from '../support/pages/Main.pageObject';
 
-describe('', () => {
-  before(() => {
+const Main = new MainPageObject();
 
+describe('Main page', () => {
+  before(() => {
+    cy.visit('');
   });
 
-  it('', () => {
-
+  it('should allow user to buy a laptop', () => {
+    const Name = 'Username';
+    const Country = 'Poland';
+    const City = 'Warsaw';
+    const Card = '123';
+    const Month = 'October';
+    const Year = '2025';
+    Main.clickOnLaptop();
```

Comment:
there are a lot of pages in this test. Not only main. Create a separate class for each.